CHANGELOG.md

```diff
@@ -6,17 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Password changes (`PUT /authn/:account/password`) now produce audit events with
+  message ID `password` ([cyberark/conjur#1548](https://github.com/cyberark/conjur/issues/1548))
+- API key rotations (`PUT /:authenticator/:account/api_key`) now produce audit events with
+  message ID `api-key` ([cyberark/conjur#1549](https://github.com/cyberark/conjur/issues/1549))
```

Comment:
Possibly incorrect nit, but... Is this a rotation?   In my head I think of rotations only as actions performed by a rotator.

Reply:
Yeah, this isn't performed by rotators, probably primarily because those are only implemented for `Secret`'s and this is a different concept in our data model (`Credentials`). 

"Rotation" is how we refer to this though: https://docs.conjur.org/Latest/en/Content/Developer/Conjur_API_Rotate_Personal_API_Key.htm